app/harvesters/scanr/scanr_references_converter.py

```diff
@@ -1,16 +1,57 @@
 from app.db.models.reference import Reference
 from app.harvesters.abstract_harvester_raw_result import AbstractHarvesterRawResult
 from app.harvesters.abstract_references_converter import AbstractReferencesConverter
+from app.harvesters.json_harvester_raw_result import (
+    JsonHarvesterRawResult as JsonRawResult,
+)
+from app.db.models.abstract import Abstract
+from app.db.models.reference import Reference
+from app.db.models.title import Title
+from app.db.models.subtitle import Subtitle
 
 
 class ScanrReferencesConverter(AbstractReferencesConverter):
     """
     Converts raw data from ScanR to a normalised Reference object
     """
 
-    async def convert(self, raw_data: AbstractHarvesterRawResult) -> Reference:
+    async def convert(self, raw_data: JsonRawResult) -> Reference:
         """
         Convert raw data from ScanR to a normalised Reference object
         :param raw_data: raw data from ScanR
         :return: Reference object
         """
+        new_ref = Reference()
+
+        json_payload = raw_data.payload
+
+        title = json_payload["_source"].get("title")
+        if title:
```

Comment:
If missing, it will return a None, and the if  condition won't be satisfied.
Empty string are considered as False, so the condition won't be satisfied aswell

Reply:
And you will accept a bibliographic reference without title that will be displayed as a blank line in user interfaces